.travis.yml

```diff
@@ -5,6 +5,7 @@ services:
 
 before_script:
   - psql -U postgres -c 'CREATE DATABASE chitter_test;'
+  - psql -U postgres -c '\i ./db/migrations/chitter_test_db.sql;'
```

Comment:
Good job! I didn't get travis fully setup so well done on powering through on this extra step!